content/en/Integrations/IntegrationBuilder/How to Guides/_index.md

```diff
@@ -15,6 +15,7 @@ weight: 30
 | Microsoft Teams   | Send notifications for findings | [Click Here](./microsoft-teams)                  |
 | Microsoft Outlook | Send notifications for findings | [Click Here](./outlook-notifications)            |
 | Google Sheets     | Import assets                   | [Click Here](./import-assets-from-google-sheets) |
+| Generic HTTP      | Send HTTP requests              | [Click Here](./http)                             |
```

Comment:
I think we can leave it out of the application specific guides table since it's not application specific

Reply:
https://github.com/cobalthq/cobalt-product-public-docs/pull/581/commits/e1da25793ab369d0fa82e308d62d2aed4fbb2071